build/repo.targets

```diff
@@ -154,22 +122,26 @@
       <Solution>
         <AdditionalProperties>RestoreGraphOutputPath=$(_RestoreGraphSpecsDirectory)%(Solution.Repository)\%(Solution.FileName)%(Solution.Extension).json</AdditionalProperties>
       </Solution>
+
+      <GraphSpecInputs Include="
```

Comment:
Not super accurate since we could emit files as part of generating the spec file, but it's better than re-running it every time now.